<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileEditor</class>
 <widget class="QMainWindow" name="FileEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
    <item>
     <widget class="QWidget" name="show_text_widget" native="true">
      <zorder>widget_btn</zorder>
      <zorder>btn_load</zorder>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="columnCount">
       <number>4</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="widget_btn">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QWidget" name="show_edit_btn" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_open">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::pressed{
	background-color: rgb(85, 85, 255);
	color: rgb(255, 255, 255);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
	border-color: rgb(28, 86, 15);
}</string>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_save">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::pressed{
	background-color: rgb(85, 85, 255);
	color: rgb(255, 255, 255);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
	border-color: rgb(28, 86, 15);
}</string>
            </property>
            <property name="text">
             <string>Save_as(F1)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_undo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::pressed{
	background-color: rgb(85, 85, 255);
	color: rgb(255, 255, 255);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
	border-color: rgb(28, 86, 15);
}</string>
            </property>
            <property name="text">
             <string>Undo(F2)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_redo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::pressed{
	background-color: rgb(85, 85, 255);
	color: rgb(255, 255, 255);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
	border-color: rgb(28, 86, 15);
}</string>
            </property>
            <property name="text">
             <string>Redo(F3)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_find">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::pressed{
	background-color: rgb(85, 85, 255);
	color: rgb(255, 255, 255);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
	border-color: rgb(28, 86, 15);
}</string>
            </property>
            <property name="text">
             <string>Find(F4)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_replace">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::pressed{
	background-color: rgb(85, 85, 255);
	color: rgb(255, 255, 255);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
	border-color: rgb(28, 86, 15);
}</string>
            </property>
            <property name="text">
             <string>Replace(F5)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_jump">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::pressed{
	background-color: rgb(85, 85, 255);
	color: rgb(255, 255, 255);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
	border-color: rgb(28, 86, 15);
}</string>
            </property>
            <property name="text">
             <string>Jump(F6)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="show_info_btn" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_info">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::checked{
     background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.498, y2:0.989, stop:0 rgba(0, 170, 255, 185), stop:1 rgba(74, 207, 255, 180));
     border: 1px solid rgb(190, 190, 190);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
}</string>
            </property>
            <property name="text">
             <string>Operation info</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_warn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::checked{
     background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.498, y2:0.989, stop:0 rgba(0, 170, 255, 185), stop:1 rgba(74, 207, 255, 180));
     border: 1px solid rgb(190, 190, 190);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
}</string>
            </property>
            <property name="text">
             <string>Warn info</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_err">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::checked{
     background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.498, y2:0.989, stop:0 rgba(0, 170, 255, 185), stop:1 rgba(74, 207, 255, 180));
     border: 1px solid rgb(190, 190, 190);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
}</string>
            </property>
            <property name="text">
             <string>Error info</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="show_main_btn" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_load">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::pressed{
	background-color: rgb(85, 85, 255);
	color: rgb(255, 255, 255);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
	border-color: rgb(28, 86, 15);
}</string>
            </property>
            <property name="text">
             <string>SaveLoad</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_cancel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton::pressed{
	background-color: rgb(85, 85, 255);
	color: rgb(255, 255, 255);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
	border-color: rgb(28, 86, 15);
}</string>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionFind"/>
   <addaction name="actionReplace"/>
   <addaction name="actionJump"/>
   <addaction name="separator"/>
   <addaction name="actionLoad"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/document-open.png</normaloff>:/new/prefix1/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/new/prefix1/images/document-save-as.png</normaloff>:/new/prefix1/images/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save_as</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/new/prefix1/images/edit-undo.png</normaloff>:/new/prefix1/images/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/new/prefix1/images/edit-redo.png</normaloff>:/new/prefix1/images/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>SaveLoad</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/new/prefix1/images/edit-find.png</normaloff>:/new/prefix1/images/edit-find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;find</string>
   </property>
   <property name="iconText">
    <string>&amp;find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/new/prefix1/images/edit-find-replace.png</normaloff>:/new/prefix1/images/edit-find-replace.png</iconset>
   </property>
   <property name="text">
    <string>&amp;replace</string>
   </property>
   <property name="iconText">
    <string>&amp;replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionJump">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/new/prefix1/images/document-send.png</normaloff>:/new/prefix1/images/document-send.png</iconset>
   </property>
   <property name="text">
    <string>jump</string>
   </property>
   <property name="iconText">
    <string>&amp;jump</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
